workflow/tests/hpxml_translator_test.rb

```diff
@@ -490,14 +480,73 @@ def _get_sizing_results(runner)
   end
 
   def _verify_simulation_outputs(runner, rundir, hpxml_path, results)
-    # Check that eplusout.err has no lines that include "Blank Schedule Type Limits Name input"
-    # Check that eplusout.err has no lines that include "FixViewFactors: View factors not complete"
+    # Check for unexpected warnings
     File.readlines(File.join(rundir, 'eplusout.err')).each do |err_line|
+      next unless err_line.include? '** Warning **'
+
+      # General
       next if err_line.include? 'Schedule:Constant="ALWAYS ON CONTINUOUS", Blank Schedule Type Limits Name input'
       next if err_line.include? 'Schedule:Constant="ALWAYS OFF DISCRETE", Blank Schedule Type Limits Name input'
+      next if err_line.include? 'Output:Meter: invalid Key Name'
+      next if err_line.include? 'GetSurfaceData: Very small surface area'
+      next if err_line.include? 'Entered Zone Volumes differ from calculated zone volume'
+      next if err_line.include?('CalculateZoneVolume') && err_line.include?('not fully enclosed')
+      next if err_line.include?('GetInputViewFactors') && err_line.include?('not enough values')
+      next if err_line.include? 'Pump nominal power or motor efficiency is set to 0'
+      next if err_line.include? 'volume flow rate per watt of rated total cooling capacity is out of range'
+      next if err_line.include? 'volume flow rate per watt of rated total heating capacity is out of range'
+      next if err_line.include? 'The following Report Variables were requested but not generated'
+      next if err_line.include? 'Timestep: Requested number'
+      next if err_line.include? 'The Standard Ratings is calculated for'
+      next if err_line.include?('CheckUsedConstructions') && err_line.include?('nominally unused constructions')
+      next if err_line.include?('GetDXCoils: Coil:Heating:DX') && err_line.include?('curve values') # Defrost curve
+      next if err_line.include? 'Full load outlet air dry-bulb temperature < 2C. This indicates the possibility of coil frost/freeze.' # HPWH located outside, these warnings are fine
+      next if err_line.include? 'Full load outlet temperature indicates a possibility of frost/freeze error continues.' # HPWH located outside, these warnings are fine
+      next if err_line.include?('WetBulb not converged after') && err_line.include?('iterations(PsyTwbFnTdbWPb)')
+
+      # TODO: Eliminate these warnings?
+
+      # GSHP:
+      next if err_line.include? 'Borehole shank spacing is less than the pipe diameter. U-tube spacing is reference from the u-tube pipe center.'
+      next if err_line.include? 'Shank spacing is set to the outer pipe diameter.'
+
+      # DHW Combi:
+      next if err_line.include? 'Missing temperature setpoint for LeavingSetpointModulated mode'
```

Comment:
This occurred to any input file containing boilers. I tried to attach a temperature setpoint manager to boiler component, but it messed up the PlantEquipmentOperationSchemes for boiler plant loop when it's connected to a combi plant loop(not sure if it will mess up those without combi systems). I tried to track down to where we assign PlantEquipmentOperationSchemes but no luck. Any thought? Btw, I think this issue is not concerning because the warning itself says simulation continues with plant loop setpoint (where we implemented boiler setpoint) applied to boiler component.

Reply:
It's possible that OpenStudio is automatically assigning the PlantEquipmentOperationSchemes object during forward translation?

If it's not concerning, you can move this line to above the "# TODO: Eliminate these warnings?" line. And add a simple comment explaining why it's not concerning (like I did for the "HPWH located outside" warnings).